.github/workflows/abi.yaml

```diff
@@ -8,11 +8,9 @@
 # in a new minor postgresql version.
 name: ABI Test
 on:
-  schedule:
-    # run daily 20:00 on main branch
-    - cron: '0 20 * * *'
```

Comment:
Since the purpose of this is to detect incompatible upstream changes we might wanna keep it on schedule and not triggered by commit in our repo